docs/knowledgebase/integrate/subkey.md

```diff
@@ -163,32 +159,32 @@ You can inspect password-protected keys either by passing the `--password` flag
 the end of the mnemonic:
 
 ```bash
-$ subkey -p "pencil laptop kitchen cutter" inspect "image stomach entry drink rice hen abstract moment nature broken gadget flash"
+$ subkey inspect-key --uri "image stomach entry drink rice hen abstract moment nature broken gadget flash" --password "pencil laptop kitchen cutter"
 Secret phrase `image stomach entry drink rice hen abstract moment nature broken gadget flash` is account:
   Secret seed:      0x11353649ecfb97ed44aa4b3516c646fa49ecd3f71cc0445647a25379f848b336
   Public key (hex): 0x1641450068c6ed825726c2a854d830352b75fb1a05e6fbcc9bc47a398a581a29
   Account ID:       0x1641450068c6ed825726c2a854d830352b75fb1a05e6fbcc9bc47a398a581a29
   SS58 Address:     5CZtJLXtVzrBJq1fMWfywDa6XuRwXekGdShPR4b8i9GWSbzB
 
-$ subkey inspect "image stomach entry drink rice hen abstract moment nature broken gadget flash///pencil laptop kitchen cutter"
+$ subkey inspect-key --uri "image stomach entry drink rice hen abstract moment nature broken gadget flash///pencil laptop kitchen cutter"
 Secret Key URI `image stomach entry drink rice hen abstract moment nature broken gadget flash///pencil laptop kitchen cutter` is account:
   Secret seed:      0x11353649ecfb97ed44aa4b3516c646fa49ecd3f71cc0445647a25379f848b336
   Public key (hex): 0x1641450068c6ed825726c2a854d830352b75fb1a05e6fbcc9bc47a398a581a29
   Account ID:       0x1641450068c6ed825726c2a854d830352b75fb1a05e6fbcc9bc47a398a581a29
   SS58 Address:     5CZtJLXtVzrBJq1fMWfywDa6XuRwXekGdShPR4b8i9GWSbzB
 ```
 
-Let's say you want to use the same private key on Kusama network. Use `-n` to get your address
-formatted for Kusama. Notice that the public key is the same, but the address has a different
-format.
+Let's say you want to use the same private key on Kusama network. Use `--network` to get your
+address formatted for Kusama. Notice that the public key is the same, but the address has a
+different format.
 
 ```bash
-$ subkey -n kusama inspect "spend report solution aspect tilt omit market cancel what type cave author"
+$ subkey inspect-key --uri "spend report solution aspect tilt omit market cancel what type cave author" --network kusama
```

Comment:
Ugh, after the mnemonic works but it's gross.
```suggestion
$ subkey inspect-key --network kusama "spend report solution aspect tilt omit market cancel what type cave author"
```

Reply:
how do you make this sexy and not gross @joepetrowski

Reply:
By committing my suggestion :)